Shared/src/Datahub.Shared/Datahub.Shared.projitems

```diff
@@ -15,7 +15,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Entities\TerraformWorkspace.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Entities\TerraformTemplate.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Enums\TerraformRole.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Exceptions\ProjectNoFoundException.cs" />
+    <Compile Include="..\..\..\Shared\src\Datahub.Shared\Exceptions\ProjectNotFoundException.cs" />
```

Comment:
This doesn't feel right @ErikApption, was that added by VS?

Reply:
`<Compile Include="$(MSBuildThisFileDirectory)Exceptions\ProjectNoFoundException.cs" />`

Reply:
fixed